stats.js

```diff
@@ -41,15 +44,22 @@ export var Stats = function () {
 
 		update: function(webGLRenderer){
 			frames++;
-		
-	        if (Date.now() > lastTime + 1000) {
 
+			var i = 0;
 
-				var i	= 0;
-				msTexts[i++].textContent = "FPS: "	+ Math.round((frames * 1000) / (Date.now() - lastTime));
+			// Update every frame
+			if(localPlayer) {
```

Comment:
Is this update called even when the stats panel is closed? This has allocations and isn't free so I don't want to take away CPU from the app by default.